test/elevation.c

```diff
@@ -0,0 +1,23 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Nowadays the first line usually looks like [this](https://github.com/libuv/libuv/blob/v1.x/test/test-udp-alloc-cb-fail.c#L1)